packages/azure-themes/src/azure/Constants.ts

```diff
@@ -5,25 +5,22 @@ import {
   getScreenSelector,
 } from '@fluentui/react/lib/Styling';
 
-export const inputHeight = '18px';
-export const commandBarHeight = '36px';
-export const buttonPadding = '0px 16px';
-export const borderWidth = '1px';
+export const borderNone = 'none';
 export const borderRadius = '3px';
-export const borderWidthError = '1px';
 export const borderSolid = 'solid';
-export const borderNone = 'none';
+export const borderWidth = '1px';
+export const buttonPadding = '0px 16px';
+export const choiceFieldHeight = '18px';
+export const commandBarHeight = '36px';
 export const dropDownItemHeight = '32px';
 export const fontFamily =
   // eslint-disable-next-line @fluentui/max-len
   'Segoe UI, "Segoe UI Web (West European)", "Segoe UI", -apple-system, BlinkMacSystemFont, Roboto, "Helvetica Neue",sans-serif';
 export const fontWeightRegular = '400';
-export const fontWeightBold = '700';
 export const inputControlHeight = '24px';
-export const choiceGroupDimensions = '18px';
-export const inputControlPadding = '12px';
 export const inputControlHeightInner = '20px';
-export const textAlignCenter = 'center';
-export const transparent = 'transparent';
+export const inputControlPadding = '12px';
 export const MinimumScreenSelector = getScreenSelector(0, ScreenWidthMaxSmall);
 export const MediumScreenSelector = getScreenSelector(ScreenWidthMinMedium, ScreenWidthMaxMedium);
```

Comment:
These 2 should be swapped if you want to maintain alphabetical order.

```suggestion
export const MediumScreenSelector = getScreenSelector(ScreenWidthMinMedium, ScreenWidthMaxMedium);
export const MinimumScreenSelector = getScreenSelector(0, ScreenWidthMaxSmall);
```